src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs

```diff
@@ -244,7 +244,9 @@ impl HintProcessor for BuiltinHintProcessor {
             hint_code::BLAKE2S_COMPUTE => {
                 compute_blake2s(vm, &hint_data.ids_data, &hint_data.ap_tracking)
             }
-            hint_code::VERIFY_ZERO => verify_zero(vm, &hint_data.ids_data, &hint_data.ap_tracking),
+            hint_code::VERIFY_ZERO | hint_code::VERIFY_ZERO_WHITELIST => {
```

Comment:
They are both "whitelist". I'd rather rename both to something like `_V1` and `_V2` if there's no other way to distinguish.

Reply:
Agreed. I was naming them like this because I thought the others weren't on the whitelist, but some of them are bound to collide anyways.

Reply:
Yes, everything that we support is part of at least one whitelist.